### DISCUSSION TITLE
Quarto (maths) to Excalidraw

### DISCUSSION THREAD

**Discussion by baptiste** [quarto-dev/quarto-cli#4144] [Show and Tell]
I thought I'd share this writeup here, in case someone with Lua experience fancies the challenge of writing a custom writer :)

blog post: https://tinyurl.com/ba-quarto-excalidraw
test repo: https://github.com/baptiste/quarto-excalidraw/

<img width="1306" alt="Screenshot 2023-01-30 at 11 00 43" src="https://user-images.githubusercontent.com/18970/215357985-0230daaf-77a7-49fc-8bdd-2dd7469a5142.png">

**Comment by cscheid** [comment]
We're going to have at least one substantial custom writer format in the 1.3 release, so you'll have an example to draw from. Keep an eye out for our Confluence custom format coming up in the next few weeks.

**Comment by andrewpbray** [comment]
I just came upon this post while thinking about the same thing (interoperability between excalidraw and quarto) and I love your solution - nice work!

**Comment by jimjam-slam** [comment]
[Obsidian](https://obsidian.md) has a JSON-based [Canvas](https://obsidian.md/canvas) format that might also make for an interesting custom writer!

I don't use Obsidian myself (although I'm thinking of migrating from Notion...), but I've seen a few folks express interest in Quarto/Obsidian workflows. Lots of different potential workflows, though...
